pkg/management/extensions.go

```diff
@@ -316,7 +316,44 @@ func newHandler(
 				status.Errorf(codes.InvalidArgument, err.Error()))
 			return
 		} else if len(body) > 0 {
-			if err := reqMsg.UnmarshalMergeJSON(body); err != nil {
+			var err error
+			bodyFieldPath := rule.GetHttp().GetBody()
+			if bodyFieldPath == "" {
+				bodyFieldPath = "*"
+			}
```

Comment:
Can you clarify this? Does this mean all protos that do not have the `google.http.api.body` options set will have it implicitly set as `google.http.api.body : *`?

Reply:
Yeah, this matches the logic in grpc-gateway (but notably does not match the logic in upstream google apis)

Reply:
this seems side-effecty, are there some edge cases we can add to the tests? also we should verify this doesn't effect consumers of the api like the dashboard

Reply:
To clarify, this is the same as the previous implementation. from grpc-gateway:
```
func (b binding) GetBodyFieldPath() string {
	if b.Body != nil && len(b.Body.FieldPath) != 0 {
		return b.Body.FieldPath.String()
	}
	return "*"
}
```